get_libwallet_api.sh

```diff
@@ -1,60 +1,81 @@
 #!/bin/bash
 
 
-BITMONERO_URL=https://github.com/monero-project/bitmonero.git
-BITMONERO_BRANCH=master
+MONERO_URL=https://github.com/monero-project/monero.git
+MONERO_BRANCH=master
+# MONERO_URL=https://github.com/mbg033/monero.git
```

Comment:
Just a note for others trying to compile. Uncomment these 2 lines or wait for [#1136](https://github.com/monero-project/monero/pull/1136) to be merged.